apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ResponseField.kt

```diff
@@ -17,7 +17,8 @@ class ResponseField(
     val responseName: String,
     val fieldName: String,
     val arguments: Map<String, Any?>,
-    val conditions: List<Condition>
+    val conditions: List<Condition>,
+    val possibleFieldSets: Map<String, Array<ResponseField>>,
```

Comment:
just to clarify: this called `possible*` because of directives and fragments?

Reply:
I can drop the `possible*`, it was mainly because of different type conditions but this just calling that `fieldSets` is simpler.